README.md

```diff
@@ -16,11 +16,13 @@ See the [wiki](https://github.com/richardwilly98/elasticsearch-river-mongodb/wik
 
 In order to install the plugin, simply run: ```bin/plugin --install com.github.richardwilly98.elasticsearch/elasticsearch-river-mongodb/2.0.9```
 
+[El Past Releases](https://www.elastic.co/downloads/past-releases)
```

Comment:
it is done please accept the pull

On Wed, Mar 16, 2016 at 1:51 AM, Ben McCann <notifications@github.com>
wrote:

> In README.md
> <https://github.com/richardwilly98/elasticsearch-river-mongodb/pull/578#discussion_r56209084>
> :
>
> > @@ -16,11 +16,13 @@ See the [wiki](https://github.com/richardwilly98/elasticsearch-river-mongodb/wik
> >
> >  In order to install the plugin, simply run: ```bin/plugin --install com.github.richardwilly98.elasticsearch/elasticsearch-river-mongodb/2.0.9```
> >
> > +[El Past Releases](https://www.elastic.co/downloads/past-releases)
>
> Yes, I know. I wasn't suggesting changing the link url. Only the link
> text, which is confusing
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/richardwilly98/elasticsearch-river-mongodb/pull/578/files/20f3c1f560801aeb8610ef41a19ed6cac6ef4473#r56209084>
>



-- 

   1.

*Ramkumar**, രാംകുമാര്‍, ராம்குமார்*